applications/FemToDemApplication/custom_processes/compute_initial_volume_process.cpp

```
@@ -65,6 +65,7 @@ double ComputeInitialVolumeProcess::ComputeInitialVolumeSubModel(
         return Globals::Pi * std::pow(max_distance, 2) * 0.25;
     } else { // 3D
         // TODO
```

I'd throw an error in here.